tensorflow_datasets/image/cityscapes_test.py

```diff
@@ -0,0 +1,27 @@
+
+'''Tests for Cityscapes dataset module.'''
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from tensorflow_datasets import testing
+from tensorflow_datasets.image import cityscapes
+
+class CityscapesTest(testing.DatasetBuilderTestCase):
+  DATASET_CLASS = cityscapes.Cityscapes
+  SPLITS = {  # Expected number of examples on each split.
+      'train': 3,
+      'validation': 1,
+      'test': 2,
+  }
+
+  DATASET_CLASS = cityscapes.CityscapesCoarse
```

Comment:
does it work? I think that you might need to create another class in this file (otherwise you might be just overwriting the entry from line 12)

Reply:
Yes this is incorrect! I wasn't sure how to run the unit tests until today and I need another class.